docs/sources/reference/commandline/cli.md

```diff
@@ -149,6 +149,7 @@ expect an integer, and they can only be specified once.
       --default-gateway-v6=""                Container default gateway IPv6 address
       --dns=[]                               DNS server to use
       --dns-search=[]                        DNS search domains to use
+      --default-ulimit=[]                    Set default ulimit settings for containers
```

Comment:
We can fix whatever is left over ;) I say merge

On Thursday, May 7, 2015, Arnaud Porterie <notifications@github.com> wrote:

> In docs/sources/reference/commandline/cli.md
> <https://github.com/docker/docker/pull/12165#discussion_r29868265>:
>
> > @@ -149,6 +149,7 @@ expect an integer, and they can only be specified once.
> >        --default-gateway-v6=""                Container default gateway IPv6 address
> >        --dns=[]                               DNS server to use
> >        --dns-search=[]                        DNS search domains to use
> > +      --default-ulimit=[]                    Set default ulimit settings for containers
>
> Alphabetical sort (I have OCD)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/docker/docker/pull/12165/files#r29868265>.
>